plugins/browser-plugin-media-tracking/src/findMediaElement.ts

```diff
@@ -25,6 +25,10 @@ function findMediaElementChild(el: Element): SearchResult {
     if (elem.length === 1) {
       if (isAudioElement(elem[0])) return { el: elem[0] };
       if (isVideoElement(elem[0])) return { el: elem[0] };
+    } else if (elem.length === 2 && tag === 'VIDEO' && isVideoElement(elem[0])) {
```

Comment:
If we want to make it really specific we can check for classnames in the container containing the string `jwplayer` etc.

Reply:
This _might_ catch other players similar to jwplayer so I'd probably leave it generic and say a little prayer. Can always tighten it up if this leads to unexpected consequences for other players, seems pretty safe tho.